### ISSUE TITLE
Things after lists start breaking

### ISSUE THREAD

**Issue by tristanjuricek** [tristanjuricek/knockoff#13]
OK I had an embedded list, and this seemed to cause a parsing error:

```
* This is a long line that wrapped
  **bold**
```

And then I notice that if you want a code block that trails a list, things are not so happy:

```
1. List item

    code block
```

That should actually be a code block within a complex list.

**Comment by tristanjuricek** [comment]
No, wait, it should just be the next paragraph in a complex list.

**Comment by tristanjuricek** [comment]
Most of the list issues were fixed in 0.5.0, there may be others... we'll see.
